src/Tests/dotnet.Tests/ParserTests/MSBuildArgumentCommandLineParserTests.cs

```diff
@@ -31,9 +31,10 @@ public void MSBuildArgumentsAreForwardedCorrectly(string[] arguments, bool build
             RestoringCommand command = buildCommand ? 
                 BuildCommand.FromArgs(arguments) : 
                 PublishCommand.FromArgs(arguments);
+            var expectedArguments = arguments.Select(a => a.Replace("-p:", "-property:"));
```

Comment:
we expand the `-p` form to `-property` now for consistency, so the expectations needed updating